Actions-Runner-Controller-Overview.md

```diff
@@ -0,0 +1,128 @@
+## Introduction
+This document provides a high level overview of Actions Runner Controller (ARC). ARC enables running Github Actions Runners on Kubernetes(K8s) clusters.
+
+This document provides a background of Github Actions, Self-hosted Runners and overview of ARCis also provided. By the end of the doc, the reader is expected to have a good understanding of ARC, setup and try out basic scenarios and set the foundation to review other advanced topics covered outside of the doc.
+
+## GitHub Actions
+GitHub Actions is a continuous integration and continuous delivery (CI/CD) platform to automate your build, test, and deployment pipeline. 
+
+You can create workflows that build and test every pull request to your repository, or deploy merged pull requests to production. Your workflow contains one or more jobs which can run in sequential order or in parallel. Each job will run inside its own runner and has one or more steps that either run a script that you define or run an action, which is a reusable extension that can simplify your workflow. To learn more about about Actions - see "[Github Actions](https://docs.github.com/en/actions/learn-github-actions)."
+
+## Runners
+Runners execute the job that is assigned to them by Github Actions workflow. There are two types of Runners.. 
```

Comment:
```suggestion
Runners execute the job that is assigned to them by Github Actions workflow. There are two types of Runners.
```

Reply:
Thanks foe the suggestion. Actually fixed this in latest commit by replacing `..` with  ':`